### ISSUE TITLE
Disable accelerator for specific menu item roles

### ISSUE THREAD

**Issue by strikeflare** [electron/electron#12384]
* Electron version: 1.8.4
* Operating system: win32 (Windows 10)

### Expected behavior
I have a case where I should disable some menu shortcuts (accelerators). For example, I defined a menu role "reload" and disabled the shortcut ("Ctrl+R") as shown below. As a result, the click behavior of this menu item is the same. But he doesn't listen to "Ctrl+R" anymore and he doesn't show the shortcut next to the label. This has worked fine in version 1.7.11.
![expected](https://user-images.githubusercontent.com/2511622/37760322-95d361d0-2db6-11e8-87c7-beb69552c23f.jpg)

### Actual behavior
In the latest version, the way how I disabled the accelerators doesn't work anymore. The shortcut is still displayed next to the label and still listens to the input events (for example "Ctrl+R"). 

![actual](https://user-images.githubusercontent.com/2511622/37760330-9dda4100-2db6-11e8-9373-0ab5e84d4af5.jpg)

### How to reproduce
Basically, I disabled shortcuts for some menu items as follows:

```
const template = [{
      label: 'View',
      submenu: [{
        role: 'reload',
        accelerator: '' // disable accelerator
      }]
    }];
```
I uploaded a full example [here](https://github.com/strikeflare/electron-quick-start).  

As I mentioned, this has worked in version 1.7.11. Is there a way to fix this or is there another way how to disable accelerators?

**Comment by codebytere** [comment]
i'll try to figure this out this week

**Comment by strikeflare** [comment]
Hi @codebytere . As I saw in #12449 , the cause were found and this issue seemt to be fixed. Do you know, what version and when I can expect getting this fix? Thanks a lot for this.

**Comment by codebytere** [comment]
@strikeflare i've backported it to `1-8-x`, so the next release in that line will contain the fix :)

**Comment by strikeflare** [comment]
@codebytere thank you for the quick fix. 👍 Can't wait to the next update :)
